### ISSUE TITLE
Reset web password with local key

### ISSUE THREAD

**Issue by m0se** [keybase/keybase-issues#2744]
I haven't been using keybase in a while and have forgotten my "web password". So I would like to reset my password, but the following statement confuses me a bit:

>If you have Keybase installed on a computer and are still logged in, it is recommended you run this:
>...
>Otherwise, use the form below, and we can email you a reset link. You will have to start all over with new keys. If you're using the Keybase filesystem, you will lose data.

When I setup my account I never shared a PGP private key with keybase, I think I signed some stuff with my key I already had to create my account and still have access to all my keys. 

Why would I have to start over with new keys or lose anything?

Or is this message just misleading for people not hosting private keys at keybase? I have done successful password resets earlier this year without having to start over in any way, did something change in this area since last April (my last reset)?

Anyhow, if the "start over" thing does not apply to my situation, that might should be mentioned in the reset dialogue.

Thanks

**Comment by maxtaco** [comment]
Keybase uses the same passphrase to: (1) authenticate users to the Web site; (2) encrypt PGP private keys for those who choose to upload (which you didn't); and (3) to lock local copies of your "device specific secret keys", which never leave your devices.  KBFS encrypts data for your device-specific keys.  So if you lose your password, you lose the ability to unlock those keys, and therefore, you might lose data. Since we can't decrypt the data, or decrypt your device-specific private key, the only thing to do is to reset your account in this case.

**Comment by m0se** [comment]
Ah ok, thanks. I did the reset and everything is fine. I interpreted "starting over with a new" key as I would have to change to a different key re-do the connecting of all other accounts again.

Maybe the reset dialogue could be more clear on that like adding "You will have to start all over with new keys, **if you have no access to your private keys**"

I don't know maybe I'm using keybase wrong and it's very exotic for keybase users to maintain there main keys indipendant of keybase, but for me it was part of what made keybase interesting.
